zos_basics/constructs/inventories/host_vars/zos_host.yml

```diff
@@ -1,23 +1,53 @@
 ---
 ################################################################################
-# Copyright (c) IBM Corporation 2021
+# Copyright (c) IBM Corporation 2021, 2024
 ################################################################################
 
 ################################################################################
 # Description of the properties used in this configuration:
 # - Property `PYZ` is the python installation home path on the z/OS managed node (target),
-#    e.g, pyz: "/usr/lpp/IBM/cyp/v3r8/pyz"
+#    e.g, pyz: "/usr/lpp/IBM/cyp/v3r12/pyz"
+# - Property `PYZ_VERSION` is the version of python on the z/OS managed node (target),
+#    e.g, pyz_version: "3.12"
 # - Property `ZOAU` is the ZOAU installation home on the z/OS managed node (target),
 #    e.g, zoau: "/usr/lpp/IBM/zoautil"
-# - property `ansible_python_interpreter` is the z/OS managed node (target) Python
+# - Property `ZOAU_PYTHONPATH` is the path to the ZOA Utilities python library.
+#    Specifically, the path points to the directory containing `zoautil_py`.
+#    If the wheel installation method is used with the `--target` or `--user` option,
+#   `ZOAU_PYTHONPATH` should point to the directory the wheel file was installed to.
+#    e.g ("/usr/zoau-wheel/")
+#    Else `ZOAU_PYTHONPATH` should point to the directory containing the pre-compiled binary.
+#    For ZOAU v1.2.x, the pre-compiled binary is located in the `lib` directory.
+#    e.g ("/usr/lpp/IBM/zoautil/lib" or "{{ ZOAU }}/lib")
+#    For ZOAU v1.3.x, the pre-compiled binary is located in a sub-directory
+#    corresponding to the `PYZ_VERSION` inside the `lib` directory,
+#    e.g ("/usr/lpp/IBM/zoautil/lib/3.12" or "{{ ZOAU }}/lib/{{ PYZ_VERSION }}")
+# - Property `ansible_python_interpreter` is the z/OS managed node (target) Python
 #    binary installation path, e.g, ansible_python_interpreter: "{{PYZ}}/bin/python3"
 #
-# Note, PYZ and ZOAU environment variables must be configured.
+# Note, PYZ, PYZ_VERSION, and ZOAU environment variables must be configured.
 ################################################################################
 
 PYZ: "path_to_python_installation_on_zos_target"
+PYZ_VERSION: "3.xx"
+
 ZOAU: "path_to_zoau_installation_on_zos_target"
 
+# Configuring the `ZOAU_PYTHONPATH` property is required.
+
+# If ZOAU installed by wheel with the --target or --user option, uncomment the following line:
+# ZOAU_PYTHONPATH: "{{ path_to_wheel_installation_directory }}"
+
+# If ZOAU is installed to the default location used by pip, the PYTHONPATH environment variable
```

Comment:
`pip` can also install a `*.whl` as well, so saying pip here can be misleading.

Maybe:
```
If the ZOAU python zoautil_py package has been installed from the source distribution (pip3 install *.tar.gz) the PYTHONPATH environment variable does not need to be configured for the ZOAU python library to be found. Uncomment the following line:
ZOAU_PYTHONPATH: ""
```

Reply:
I see the confusion here. I think your suggestion also does not quite capture the issue. 

for context - python, by default, checks for python packages in a set of default locations. The 'pythonpath' environment var tells python to go look in other path(s) too, specifically in those defined in the env var. 

the phrase i used "to the default location used by pip" was meant to be in reference of those default locations python automatically checks for python packages.

Wonder if we can come up with a clearer way to get the message across

Reply:
When I read it the first time , my eyes navigated towards "pip" , overlooking ""to the default location", so your right, my suggestion only partly addresses this. Ill take a crack at it;

```
If the ZOAU python zoautil_py package has been installed in the default location that is part of Pythons search path, the `ZOAU_PYTHONPATH` environment variable must be an empty string. Packages installed using `pip` generally default to  the `site-packages` directory within the Python installation, eg `/usr/lpp/IBM/cyp/v3r12/pyz/lib/python3.12/site-packages/`, the installation uses options such as `--user` or `--target` a non-default location will contain the module which must then be configured for `ZOAU_PYTHONPATH`.


I don't know it adds value or if desirable to squeeze this in , but python 1 liner will print Python locations as part of the search pattern.
`python -c "import sys;print('\n'.join(sys.path))"` and `pip3 list -v` will show where Python modules are isntalled.